projects/netgrif-components-core/src/lib/view/workflow-view/abstract-workflow-view.component.ts

```diff
@@ -29,8 +30,9 @@ export abstract class AbstractWorkflowViewComponent extends AbstractViewWithHead
     protected constructor(protected _sideMenuService: SideMenuService,
                           protected _workflowViewService: WorkflowViewService,
                           protected _log: LoggerService,
-                          protected _processService: ProcessService) {
-        super(_workflowViewService);
+                          protected _processService: ProcessService,
+                          protected _activatedRoute?: ActivatedRoute) {
```

Comment:
protected ?

Reply:
Every dependency was protected, so I decided to give it a protected prefix. If you would like to, I can remove that.

Reply:
if an access modifier is specified it creates an attribute for this class that shadows/overrides the attribute in the parent. Honestly I am not sure how typescript handles this, but I am uncomfortable living with this uncertainity. I am aware that there is a lot of this syntax in our code-base, but I think we should be attempting to remove it over time, so that the access modifier is only used at the "lowest" level where the attribute is defined.

Since you are adding a new injection dependency that is just passed to the super class it should not specify an access modifier, as the super class (transitively) declares the attribute.

It probably does not matter, since it seems to work either way, but I just don't like it. You can leave it if you preffer.

Reply:
Thanks for the explanaiton! I actually just read the same thing in TS docs, I aggree with you, it is now removed!